ros/angel_datahub/src/angel_datahub.cpp

```diff
@@ -168,10 +168,15 @@ ::AngelDataHub( rclcpp::NodeOptions const& options )
   // This two-stage declare->get allows the lack of passing a parameter to
   // throw an error with the parameter name in the error so the user has a
   // clue what is going wrong.
-  declare_parameter< std::string >( PARAM_TCP_SERVER_ADDR );
-  declare_parameter< std::string >( PARAM_TOPIC_INPUT_DET_3D );
-  declare_parameter< std::string >( PARAM_TOPIC_OUTPUT_HEADSET_POSE );
-  declare_parameter< std::string >( PARAM_TOPIC_INPUT_TASK_UPDATE );
+  declare_parameter( PARAM_TCP_SERVER_ADDR );
+  declare_parameter( PARAM_TOPIC_INPUT_DET_3D );
+  declare_parameter( PARAM_TOPIC_OUTPUT_HEADSET_POSE );
+  declare_parameter( PARAM_TOPIC_INPUT_TASK_UPDATE );
```

Comment:
Why are we dropping the types?

Reply:
@Purg might have insight on this

Reply:
@periakiva After testing with BBN can you see if adding the types back in breaks anything? I really don't want to drop them if we don't have to

Reply:
I am not sure why this was commented out. maybe @Purg knows

Reply:
Sanity checked with @Purg that this is intentional